concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -99,23 +65,54 @@ def get_force_get_task_result(task_id, subtask_id, current_time, cluster_consts,
     return force_get_task_result
 
 
+def upload_file_to_storage_cluster(file_content, file_path, upload_token):
+    dumped_upload_token = shortcuts.dump(upload_token, None, CONCENT_PUBLIC_KEY)
+    b64_encoded_token = b64encode(dumped_upload_token).decode()
+    headers = {
+        'Authorization': 'Golem ' + b64_encoded_token,
+        'Concent-Auth': b64encode(
+            create_client_auth_message(PROVIDER_PRIVATE_KEY, PROVIDER_PUBLIC_KEY, CONCENT_PUBLIC_KEY)).decode(),
+        'Concent-upload-path': file_path,
+        'Content-Type': 'application/octet-stream'
+    }
+    return requests.post(
+        "{}".format(STORAGE_CLUSTER_ADDRESS + 'upload/'),
```

Comment:
For better readability you can do:
"{}upload/".format(STORAGE_CLUSTER_ADDRESS),